systems/WikibaseRepo/05-Building_Block_View.md

```diff
@@ -79,9 +79,9 @@ TBA
 
 | Building Block                              | Type/Context        | Responsibility                                                                                                                                                                                               |
 | ------------------------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| SpecialWikibasePage                         | base class          | Base class for Wikibase Special Pages                                                                                                                                                                        |
+| SpecialWikibasePage                         | base class          | Base class for Wikibase [Special Pages](../overview/12-Glossary.md#special-page)                                                                                                                             |
 | EntityIdParser                              | Handling user input | Interface for objects that can parse strings into EntityIds                                                                                                                                                  |
-| TokenCheckInteractor                        | Handling user input | Checks a submitted edit token                                                                                                                                                                                 |
+| TokenCheckInteractor                        | Handling user input | Checks a submitted edit token                                                                                                                                                                                |
```

Comment:
Not sure what happened in this line?
I don't see any change?

Reply:
I think the md linter added this: I think it deleted a single space. I kept fighting against it, then gave up at some point.